### ISSUE TITLE
Throttle promise IE 11 polyfill

### ISSUE THREAD

**Issue by shpfive** [storyblok/storyblok-js-client#9]
Hello.
I have a problem using storyblok js client in Nuxt 2.3.1.

I can't get it working in ie 11 because of import of throttle-promise and used "Number.isFinite". 

Any ideas ?
thanks in advance

**Comment by onefriendaday** [comment]
Hi!
We'll take a look into the implementation and may replace it with a fork of the package throttle-promise.
In the meantime you can use the suggested polyfill from Mozilla as following:
~~~
if (Number.isFinite === undefined) Number.isFinite = function(value) {
    return typeof value === 'number' && isFinite(value);
}
~~~

**Comment by shpfive** [comment]
Thank you :)

**Comment by samuells** [comment]
Hi, any updated on this? I just hit the same problem. 😉

**Comment by onefriendaday** [comment]
Has been fixed in version `1.0.24`.
